### PR TITLE
fix: Added Missing Account Privileges

### PR DESCRIPTION
**Pull request by ChrisIsidora**
Added missing account privileges and missing view from #634 

## References
https://docs.snowflake.com/en/user-guide/security-access-control-privileges.html#global-privileges
*

### CODE REVIEW

File: pkg/resources/account_grant.go [hunk 1]
```diff
@@ -19,6 +19,13 @@ var validAccountPrivileges = NewPrivilegeSet(
 	privilegeApplyMaskingPolicy,
 	privilegeCreateShare,
 	privilegeImportShare,
+	privilegeApplyRowAccessPolicy,
+	privilegeApplyTag,
+	privilegeAttachPolicy,
+	privilegeCreateDataExchangeListing,
+	privilegeCreateNetworkPolicy,
+	privilegeOverrideShareRestrictions,
+	privilegeCreateAccount,
```
Comment (alldoami) [inline pkg/resources/account_grant.go:28]: Nit: alphabetize this list please :)

Comment (ChrisIsidora) [reply pkg/resources/account_grant.go]: Done!

**Comment by ChrisIsidora** [comment]
@alldoami Done! I merged #634  in this PR. Can you take a look again? If merged, is it possible to release a new version including these changes? Thanks in advance

**Comment by alldoami** [comment]
Oops can you fix the conflicts? 😅 I will release a new version after I merge your PR!

**Comment by ChrisIsidora** [comment]
@alldoami Done!

**Comment by ChrisIsidora** [comment]
@alldoami After merging this and updating the branch on #619 , can we check if it's also possible to merge #619 ?

File: pkg/resources/privileges.go [hunk 2]
```diff
@@ -7,43 +7,50 @@ func (p Privilege) String() string {
...
-	privilegeAddSearchOptimization  Privilege = "ADD SEARCH OPTIMIZATION"
-	privilegeApplyMaskingPolicy     Privilege = "APPLY MASKING POLICY"
-	privilegeApply                  Privilege = "APPLY"
+	privilegeSelect                    Privilege = "SELECT"
+	privilegeInsert                    Privilege = "INSERT"
+	privilegeUpdate                    Privilege = "UPDATE"
+	privilegeDelete                    Privilege = "DELETE"
+	privilegeTruncate                  Privilege = "TRUNCATE"
+	privilegeReferences                Privilege = "REFERENCES"
+  privilegeRebuild                   Privilege = "REBUILD"
```
Comment (alldoami) [inline pkg/resources/privileges.go:16]: Spacing

Comment (ChrisIsidora) [reply pkg/resources/privileges.go]: Fixed

**Comment by ChrisIsidora** [comment]
@alldoami Can you check again?

**Review by alldoami — APPROVED** [review]

**Pull request merged**